01_prepare_host.sh

```diff
@@ -100,6 +100,8 @@ function install_clusterctl() {
   curl -L https://github.com/kubernetes-sigs/cluster-api/releases/download/"${CAPIRELEASE}"/clusterctl-linux-amd64 -o clusterctl
   chmod +x ./clusterctl
   sudo mv ./clusterctl /usr/local/bin/clusterctl
+  mkdir -p home/"${USER}"/.cluster-api
+  touch home/"${USER}"/.cluster-api/version.yaml
```

Comment:
this file itself might not be needed, as long as the .cluster-api exists.

Reply:
should we ignore it maybe, if it exists, i.e `mkdir -p home/"${USER}"/.cluster-api || true `, WDYT?

Reply:
mkdir -p will not error if the folder exist. I just meant that the file "version.yaml" probably does not need to be created.